dv_utils/connectors/azure.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import copy
+import duckdb
+
+from dv_utils.connectors.connector import Configuration
+from ..log_utils import audit_log, LogLevel
+
+logger = logging.getLogger(__name__)
+
+class AZConfiguration(Configuration):
+    schema_file = "azure.json"
+    connect_type ="azure"
+    
+    description = None
+    location = None
+    file_format = None
+    encryption_key = None
+    ref_encryption_key = None
+
+    connection_key = None
+    share_access_token = None
+
+class AZConnector():
+    NAMING_CONVENTION_MODEL="{{model}}"
+
+    config: AZConfiguration
+
+    def __init__(self, config: AZConfiguration) -> None:
+        self.config = copy.copy(config)
+
+    def add_duck_db_connection(self,duckdb,ref_encryption_key: str):
+        description = self.config.description
+        location = self.config.location
+        file_format = self.config.file_format
+        encryption_key=self.config.encryption_key
+        ref_encryption_key = self.config.ref_encryption_key
+
+        connection_key=self.config.connection_key
+        share_access_token=self.config.share_access_token
```

Comment:
of these variables, only `encryption_key`, `ref_encryption_key` and `connection_key` are used. But do we really need to declare them separately if they are only used once?

Reply:
✅